src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -1,25 +1,27 @@
 package edu.kis.powp.jobs2d.drivers;
 
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.LoggerDriver;
 
 /**
  * Driver manager provides means to setup the driver. It also enables other
  * components and features of the application to react on configuration changes.
  */
 public class DriverManager {
-
-	private final Job2dDriver currentDriver = new ProxyDriver();
-
+	private Job2dDriver currentDriver = new LoggerDriver();
+	private MacroDecoratorDriver decoratorDriver = new MacroDecoratorDriver();
```

Comment:
Sorry, but I still think that at this point modifications to DriverManager won't give us any design advantage (OCP). If you still have problems to avoid that we will talk tomorrow.